application/src/main/resources/ecc.yml

```diff
@@ -24,6 +24,8 @@ connection:
     ## to connect to. The application will use the configurations
     ## specified below, connecting to the listed hosts;
     agent:
+      ## Each ecchronos instance must have unique name
```

Comment:
I think it need to be more descriptive, also, explain in the architecture.md what is that and why we need it